### PR TITLE
Rewrite of the Journey flow

### PR DESCRIPTION
**Pull request by animeshsingh**
This makes the journey more widely applicable, gets us best of both the worlds, a journey which works at starter level as well as also goes into some advanced stuff where istio sample app didnt go in its original format - win win on both sides

### CODE REVIEW

**Comment by AnthonyAmanse** [comment]
LGTM - except for diagram. `Reviews-v1` does not call the `Ratings` service. Only versions 2 and 3 of reviews service gets the Ratings

File: README.md [hunk 1]
```diff
@@ -316,6 +306,90 @@ This step shows you how to collect trace spans using [Zipkin](http://zipkin.io).
...
+
+Provision Compose for MySQL in Bluemix via https://console.ng.bluemix.net/catalog/services/compose-for-mysql  
+Go to Service credentials and view your credentials. Your MySQL hostname, port, user, and password are under your credential uri and it should look like this
+![images](images/mysqlservice.png)
+
+## 7. Modify sample application to use the external database
+
+In this step, the original sample BookInfo Application is modified to leverage a MySQL database. The modified microservices are the `details`, `ratings`, and `reviews`. This is done to show how Istio can be configured to enable egress traffic for applications leveraging external services outside the Istio data plane, in this case a database.
+
+In this step, you can eitjer choose to build your Docker images for different microservices from source in the [microservices folder](/microservices) or use the given images.
```
Comment (Tomcli) [inline README.md]: typo: `either` not eitjer

File: README.md [hunk 2]
```diff
@@ -316,6 +306,90 @@ This step shows you how to collect trace spans using [Zipkin](http://zipkin.io).
...
+
+## 8. Deploy application microservices and Istio envoys with Egress traffic enabled
+
+* Insert data in your MySQL database in Bluemix. 
+
+```bash
+$ kubectl apply -f <(istioctl kube-inject -f mysql-data.yaml --includeIPRanges=172.30.0.0/16,172.20.0.0/16)
+```
+The `--includeIPRanges` option is to pass the IP range(s) used for internal cluster services, thereby excluding external IPs from being redirected to the sidecar proxy. The IP range above is for IBM Bluemix provisioned Kubernetes Clusters. For minikube, you will have to use `10.0.0.1/24`
+> IMPORTANT NOTE: For services otuside of your cluster, you would need to enable egress traffic and add `--includeIPRanges` if it is not an http/https protocol. You can read more about enabling egress traffic for http/https protocol [here](https://istio.io/docs/tasks/egress.html#using-the-istio-egress-service)
```
Comment (Tomcli) [inline README.md]: typo: `outside` not otuside

File: README.md (hunk 1 above)
Comment (animeshsingh) [inline README.md]: done

File: README.md (hunk 2 above)
Comment (animeshsingh) [inline README.md]: done

File: GETTING_STARTED.md [hunk 3]
```diff
@@ -1,22 +1,13 @@
...
 
-The [BookInfo](https://istio.io/docs/samples/bookinfo.html) is a simple application that is composed of four microservices. The application is written in different languages for each of its microservices namely Python, Java, Ruby, and Node.js.
-
 # Prerequisite
-Create a Kubernetes cluster with either [Minikube](https://kubernetes.io/docs/getting-started-guides/minikube) for local testing, or with [IBM Bluemix Container Service](https://github.com/IBM/container-journey-template/blob/master/Toolchain_Instructions_new.md) to deploy in cloud. The code here is regularly tested against [Kubernetes Cluster from Bluemix Container Service](https://console.ng.bluemix.net/docs/containers/cs_ov.html#cs_ov) using Travis.
+Create a Kubernetes cluster with either [Minikube](https://kubernetes.io/docs/getting-started-guides/minikube) for local testing, or with [IBM Bluemix Container Service](https://github.com/IBM/container-journey-template/blob/master/Toolchain_Instructions_new.md) to deploy in cloud.
```
Comment (animeshsingh) [inline GETTING_STARTED.md]: done

**Pull request merged**